core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1165,7 +1165,7 @@ write_files:
                     done
               livenessProbe:
                 exec:
-                  command: ["/bin/bash", "-c", "(( $(date +%s) - $(stat -c%Y /probe-token) < 25*60*60 ))" ]
+                  command: ["/bin/sh", "-c", "export", "AGE=$(( $(date +%s) - $(stat -c%Y /tmp) < 25*60*60 ));  [ $AGE -gt 0 ]" ]
```

Comment:
Perhaps `export` is not strictly necessary?

Reply:
Yes you're right it works fine without `export`. Had to test various  methods since `/bin/sh` it's actually a symlinks for `/bin/dash`.